blueocean-dashboard/src/main/js/AllPipelines.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import OrganisationPipelines from './OrganisationPipelines';
+
+class AllPipelines extends Component {
+
+    render() {
+        return (
+            <OrganisationPipelines { ... this.props } />
+        );
+    }
+
+}
+
+export default AllPipelines;
```

Comment:
Should this be `OrganizationPipelines` with z not s? 

So in the end you moved back away from these higher level components?